apis/fluent-en_GB/extensions/jdk8/atrium-api-fluent-en_GB-jdk8-jvm/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/jdk8/PathAssertionsSpec.kt

```diff
@@ -2,9 +2,11 @@ package ch.tutteli.atrium.api.fluent.en_GB.jdk8
 
 import ch.tutteli.atrium.creating.Expect
 import ch.tutteli.atrium.specs.fun0
+import ch.tutteli.atrium.specs.fun1
 import java.nio.file.Path
 
 class PathAssertionsSpec : ch.tutteli.atrium.specs.integration.PathAssertionsSpec(
     fun0(Expect<Path>::exists),
-    fun0(Expect<Path>::existsNot)
+    fun0(Expect<Path>::existsNot),
+    fun1<Path, Expect<String>.() -> Unit>(Expect<Path>::fileNameWithoutExtension)
```

Comment:
I included this here but also included in PathFeatureAssertionsSpec.  How do you determine whether to put in *Spec vs *FeatureSpec?

Reply:
I put assertion functions here and shortcut for features (as yours) in PathFeatureAssertionsSpec

Reply:
I should not touch this then and put everything in PathFeatureAssertionsSpec?

Reply:
Yes please